flow/__init__.py

```diff
@@ -12,6 +12,7 @@
 from . import scheduling
 from . import errors
 from . import testing
+from .project import IgnoreConditions
```

Comment:
@vyasr Based on what you mentioned regarding __init__.py last week, I think we need this to allow user to access IgnoreConditions by `from flow import IgnoreConditions`. Is this understanding correct?

Reply:
Yes, otherwise you would have to do `from flow.project import IgnoreConditions` or similar.